internal/pkg/saelastic/utils.go

```diff
@@ -72,24 +69,21 @@ func typeSwitchAlertname(tst interface{}) (string, IndexType, error) {
 	return string(GENERICINDEX), GENERICINDEXTYPE, nil
 }
 
-// Not used
+//lint:ignore U1000 we might be able to delete this, but we should dig into the history of what it's for
```

Comment:
Well this function is definitely related to the collectd work for the connectivity, sysevent, and procevent plugins, but I have no idea why...

@aneeshkp ?